backend/tests/conftest.py

```diff
@@ -1,16 +1,45 @@
+import json
+import os
+from app.models.submission import Submission
+from app.models.user import User
 import pytest
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from fastapi.testclient import TestClient
-import os
-
+from sqlalchemy.orm import Session
+from datetime import datetime
 from app.core.database import Base
+from pyrootutils import find_root
 from app.main import app
+from fastapi.testclient import TestClient
 from app.core.session import get_db
 
 
-@pytest.fixture(scope="function")
-def test_db():
+@pytest.fixture(scope="session")
+def test_users() -> list[User]:
+    data_path = find_root() / "tests" / "test_data" / "users.json"
```

Comment:
let's not store data for test in separate jsons. It's very inconvenient, because if something isn't working, you need to go and find the file you're working with.

instead, we can have functions or fixtures. We may have say `tests/mocks/users.py` and there we have
```python
def user_1() -> dict:
    return {user_info}

def user_2() -> dict:
    ...
```
and then use it in tests like `users = [User(**user) for user in [user_1(), user_2(), user_3()]`.

Now, if you want to see what's user_1(), you can just use IDE features to jump to this function. And if you need to fix smth there, you consistently modify the python code, instead of modifying python code and jsons.

Reply:
Though, I'm fine with keeping `personal_info_extraction_debug.csv` as a file, bc I don't expect to look into it much.

Reply:
also please not call it "test_users". Anything that starts with "test_" sounds like an actual test that should be launched. You can call it "mock_users" or just "users"